block_evasys_sync.php

```diff
@@ -135,26 +136,24 @@ public function get_content() {
                         $i++;
                     }
                     $this->content->text .= html_writer::alist($outputsurveys, null, 'ol');
+                    $this->content->text .= "<fieldset>" .
```

Comment:
I may be mistaken, but if you add the inputs here, you should have one startDate / endDate input per $evasyscourseid. That would mean you can have multiple values for one name(key) in this form.